extensions/resteasy-common/deployment/src/main/java/io/quarkus/resteasy/common/deployment/ResteasyDotNames.java

```diff
@@ -85,6 +85,7 @@ public boolean test(DotName dotName) {
 
             // JAX-RS
             DotName.createSimple("javax.ws.rs.core.Response"),
+            DotName.createSimple("javax.ws.rs.core.Response.StatusType"),
```

Comment:
Now that we have `PACKAGES_IGNORED_FOR_REFLECTION` just below, I wonder if we should just ignore the root packages of all these classes.

Reply:
I'll create a follow-up PR along those lines.